doc/upload.md

```diff
@@ -0,0 +1,180 @@
+# CDI Upload User Guide
+The purpose of this document is to show how to upload a VM disk image on your local system to a PersistentVolumeClaim in Kubernetes.
+
+## Prerequesites
+You have a Kubernetes cluster up and running with CDI installed and at least one PersistentVolume is available.
+
+Commands/manifests below will be run from the root of the CDI repo against a Minikube cluster.
+
+If you are using Minikube with the `storage-provisioner` addon enabled.  You can create a PersistentVolume like so:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv0001
+spec:
+  accessModes:
+    - ReadWriteOnce
+  capacity:
+    storage: 5Gi
+  hostPath:
+    path: /data/pv0001/
+EOF
+```
+
+## Expose cdi-uploadproxy service
+In order to upload data to your cluster, the cdi-uploadproxy service must be accessible from outside the cluster.  In a production environment, the probably involves setting up a Ingress or a LoadBalancer Service.
```

Comment:
In order to upload data to your cluster, the cdi-uploadproxy service must be accessible from outside the cluster.  In a production environment, this probably involves setting up a Ingress or a LoadBalancer Service.